pkgs/by-name/ax/axmldec/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake, doxygen, boost, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "axmldec";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    repo = "axmldec";
+    owner = "ytsutano";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-LFDZZbRDa8mQmglgS4DA/OqXp0HJZ2uqg1hbStdgvUw=";
+  };
+
+  nativeBuildInputs = [ cmake doxygen ];
+  buildInputs = [ boost zlib ];
+
+  meta = with lib; {
+    mainProgram = "axmldec";
+    description = "Stand-alone binary AndroidManifest.xml decoder";
+    longDescription = ''
+      This tool accepts either a binary or a text XML file and prints the
+      decoded XML to the standard output or a file. It also allows you to
+      extract the decoded AndroidManifest.xml directly from an APK file.
```

Comment:
```suggestion
      extract the decoded {file}`AndroidManifest.xml` directly from an APK file.
```

Reply:
Not sure what this is doing but I guess it makes sense!

Reply:
Here, have a look at this: https://nixos.org/manual/nixpkgs/stable/#sec-contributing-markup

Reply:
@h7x4 this markup is not meant to be employed on meta.description, but on Markdown text files.

Indeed, nothing similar to it appears on Nix files, except in `#` comments.

`rg -n --column -i --heading --no-config -e \`\\\{.\*\\\}`

Reply:
Huh... reverted

Reply:
I see, sorry about that. Thanks!